src/webrtc/peerconnection.ts

```diff
@@ -428,6 +483,11 @@ export class PeerConnectionClass implements PeerConnection<signals.Message> {
       throw new Error('label cannot be an empty string');
     }
 
+    if (Object.keys(this.channels_).length >= this.maxChannels_) {
```

Comment:
I think this should probably also reject if `options && options.id >= this.maxChannels_`.